Source/player.cpp

```diff
@@ -2808,6 +2813,18 @@ BOOL PlrHitMonst(int pnum, int m)
 		return FALSE;
 	}
 
+#ifdef HELLFIRE
+	if (pnum < 0) {
+		adjacentDamage = TRUE;
+		pnum = -pnum;
+		if (plr[pnum]._pLevel > 20) {
+			hper -= 30;
+		} else {
+			hper -= (35 - plr[pnum]._pLevel) * 2;
+		}

```

Comment:
remove curly braces for oneliners?

Reply:
Ok :(

You can tell that I cleaned up most of player.cpp because of all the extra currlies :joy_cat: